src/Parser.elm

```diff
@@ -0,0 +1,45 @@
+module Parser exposing (..)
+
+{- Sample:
+   Compressed Twinkling Cobaltite    4397
+   Compressed Glowing Carnotite      2603
+   Glowing Cinnabar                  1918
+   Glistening Coesite                2740
+-}
+
+
+type alias OreEntry =
+    { name : String
+    , volume : Int
+    }
+
+
+parseOreInputLine : String -> Maybe OreEntry
```

Comment:
I presume that every input line is supposed to be parseable, and that it should be an error if the input is malformed. Currently though, this just skips the malformed entries silently via the "filterMap" call below.

I suggest using elm's [Result type](https://package.elm-lang.org/packages/elm/core/latest/Result), so you can return an error showing which line is malformed and why. 

Then you could display it to the user, in a little UI element for parser errors.

Reply:
Great tip ❤️🦋

In the spirit of moving quickly and breaking things, I will implement proper error checking and handling for the parser when I move to an actual parsing library.